core/spec/models/spree/address_spec.rb

```diff
@@ -119,6 +119,47 @@
     end
   end
 
+  context "when updating a record" do
+    context "if use_combined_first_and_last_name_in_address is set to false (default)" do
+      before do
+        allow(Spree::Config).to receive(:use_combined_first_and_last_name_in_address) { false }
+      end
+
+      context "and the `name` attribute is not in changeset" do
+        let(:address) { create(:address, firstname: 'John', lastname: 'Doe') }
```

Comment:
given every context includes this line, what about moving it one `context` above, so we remove the duplication?

Reply:
Sure!